### ISSUE TITLE
Top Window Bar

### ISSUE THREAD

**Issue by eniskastrati** [minbrowser/min#1921]
It would be very good if the user could move the min browser window to the next monitor(if has two displays) or just being able to move the min browser in the next workspace.
So right now the browser opens in **full screen** and you cant move it for instance to your second monitor cause its somehow static and full screen.

_And from a user perspective we are used being able to move our application windows by just going with cursor to top bar of the window holding it clicked and moving it._

**Comment by PalmerAL** [comment]
You're right this design isn't ideal, but if you unmaximize the window, a space will show up that the top that's draggable.
